mula/scheduler/server/schemas/task.py

```diff
@@ -65,3 +65,7 @@ class TaskPush(BaseModel):
     organisation: str
     priority: int | None = None
     data: dict
+
+
+class TaskPop(BaseModel):
```

Comment:
I think this new format breaks the scheduler client in the boefjes. Perhaps the `PaginatedTasksResponse` definition should be changed as well.

Reply:
Made some changes to the boefjes, would you mind checking?